lib/zoom/actions/user.rb

```diff
@@ -99,8 +99,14 @@ def user_recordings_list(*args)
 
       def user_token(*args)
         params = Zoom::Params.new(Utils.extract_options!(args))
-        params.require(:id).permit(%i[type ttl])
-        Utils.parse_response self.class.get("/users/#{params[:id]}/token", query: params.except(:id), headers: request_headers)
+        params.require(:user_id).permit(%i[type ttl])
+        Utils.parse_response self.class.get("/users/#{params[:user_id]}/token", query: params.except(:user_id), headers: request_headers)
+      end
+
+      def user_permissions(*args)
+        params = Zoom::Params.new(Utils.extract_options!(args))
+        params.require(:user_id)
+        Utils.parse_response self.class.get("/users/#{params[:user_id]}/permissions", query: params.except(:user_id), headers: request_headers)
```

Comment:
I don't think we need to send along a query.

```suggestion
        Utils.parse_response self.class.get("/users/#{params[:user_id]}/permissions", headers: request_headers)
```